tests/rptest/wasm/wasm_base_test.py

```diff
@@ -0,0 +1,78 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from rptest.wasm.wasm_test import WasmTest
+from rptest.wasm.wasm_script import WasmScript
+from rptest.wasm.wasm_build_tool import WasmTemplateRepository
+from rptest.wasm.topic import construct_materialized_topic
+
+
+class WasmBaseTest(WasmTest):
```

Comment:
nit: the naming gets a bit awkward here: WasmTest is the base of WasmBaseTest, which is the base for some wasm related tests but not others.

Would it make sense to collapse all the base functionality into WasmTest and use that across the board (it looks like with this change, WasmFilterTest is the only one that's consuming the ultimate WasmTest base class rather than WasmBaseTest)

Reply:
I think a better name would be `wasm_identity_base` or something because its job is to deploy a single `identity_coprocessor`